workspaces/adr/.changeset/calm-mirrors-mate.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage-community/search-backend-module-adr': minor
+---
+
+Initial version for the search-backend-module-adr. This module can be used to
+add the search collator directly without having to create the backend module
+on your own if using the new backend system.
```

Comment:
```suggestion
Initial version for the `@backstage-community/search-backend-module-adr`. This module can be used to add the `adr` search collator using the new backend system.
```

Reply:
Reworded the changelog. Also added a second paragraph with the types that have been moved